### DISCUSSION TITLE
lighten\darken  - doesn't work

### DISCUSSION THREAD

**Discussion by vladdoletskiy** [react-page/react-page#1211] [Q&A]
Hi, I have a problem with  lighten\darken elements on background changes. It doesn't work. Unfortunately, I use legacy MUI 4. In your demo darken works well. But lighten - not.

**Comment by vladdoletskiy** [comment]
<div style={{ display: 'flex' }}>
          <div style={{ flex: 1 }}>
            <Typography variant="body1" id="linear-gradient-darken-label">
              {this.props.translations?.darken} ({darkenFinal && (darkenFinal * 100).toFixed(0)}
              %)
            </Typography>
            <Slider
              aria-labelledby="linear-gradient-darken-label"
              value={darkenFinal}
              onChange={(e: React.SyntheticEvent, value: React.ComponentState) =>
                this.props.handleChangeDarkenPreview(value instanceof Array ? value[0] : value)
              }
              onChangeCommitted={this.props.handleChangeDarken}
              step={0.01}
              min={0}
              max={1}
            />
          </div>

          <div style={{ flex: 1, marginLeft: 16 }}>
            <Typography variant="body1" id="linear-gradient-lighten-label">
              {this.props.translations?.lighten} ({((lightenFinal as number) * 100).toFixed(0)}
              %)
            </Typography>
            <Slider
              aria-labelledby="linear-gradient-lighten-label"
              value={lightenFinal}
              onChange={(e: React.SyntheticEvent, value: React.ComponentState) =>
                this.props.handleChangeLightenPreview(value instanceof Array ? value[0] : value)
              }
              onChangeCommitted={this.props.handleChangeLighten}
              step={0.01}
              min={0}
              max={1}
            />
          </div>

**Comment by macrozone** [comment]
can you try the latest version? 5.0.2 

there it seems to work https://react-page.github.io/

there was a bug in 5.0.
